front/src/Connexion/Room.ts

```diff
@@ -34,8 +34,12 @@ export class Room {
             //We instead use 'workadventure' as a dummy base value.
             const baseUrlObject = new URL(baseUrl);
             const absoluteExitSceneUrl = new URL(identifier, 'http://workadventure/_/'+currentInstance+'/'+baseUrlObject.hostname+baseUrlObject.pathname);
-            roomId = absoluteExitSceneUrl.pathname; //in case of a relative url, we need to create a public roomId
-            roomId = roomId.substring(1); //remove the leading slash
+            if (absoluteExitSceneUrl.hostname !== 'workadventure') {
```

Comment:
shouldn't this be variable as $BASE_DOMAIN could be something else than 'workadventure'?